src/client/refresh/gl1/gl1_light.c

```diff
@@ -46,7 +46,7 @@ R_RenderDlight(dlight_t *light)
 	GLfloat vtx[3*18];
 	GLfloat clr[4*18];
 
-	unsigned int index_vtx = 4;
+	unsigned int index_vtx = 3;
```

Comment:
This change looks like it makes sense, after all the `for ( i = 0; i < 3; i++ )` loop sets a value to `vtx[0]`, `vtx[1]` and `vtx[2]`, so it makes sense that index_vtx starts at 3, so other loop first sets `vtx[3]` and the following ones - the old code apparently left `vtx[3]` uninitialized *and* wrote past the end of the array.

Would be helpful to add this information to the commit message though, because the compiler error itself isn't very specific on what's going wrong there.